docs/ota-client-guide/modules/ROOT/pages/pushing-updates.adoc

```diff
@@ -1,4 +1,4 @@
-= Upload a sample software version
+= Upload an updated software version
```

Comment:
All of the stuff on this page is about modifying a Yocto image for updating; maybe it would be better to have a title that was something like "Create and upload an updated Yocto image"?

Reply:
Well, I was trying to highlight the high-level goals in the titles. I'm only modifying the Yocto image so that I can get some new software on the device, and this happens to be the way that I have to do it (as opposed to directly installing a package). 

But we have the "Software management" topic now, so  I don't mind either way. The only other problem is that we now have many modifiers for "image"
https://www.google.com/search?q=site:docs.ota.here.com+*+image

Yocto image, disk image, software image, filesystem image, SD Card image, ect..

Perhaps "Use Yocto to update your image"?

Would be good be consistent...I know we talked about "software image" but I noticed other products talk more often about "disk images". Also, isn't Yocto a tool to BUILD images? 
In the terminology, do we meed to distinguish images built with Yocto from other types of images? Maybe the answer is "yes" do if Yocto is the tool people are using, not really sure myself.

Reply:
Looking at the actual content of the page, something like "Add software to your Yocto image" seems most appropriate. There's not a single word of instruction about uploading it; the three topics are adding a package to a yocto build, adding a layer to a yocto build, and writing your own package or layer.